### ISSUE TITLE
Fix S1450: Constant fields

### ISSUE THREAD

**Issue by extremecodetv** [SonarSource/sonar-csharp#743]
### Description

One can to accidentally change a value this variable. For prevent it i use CONST keyword. But Sonar mark it as a mistake, but it isn't.

### Repro steps

```C#
private const int foo = 10;
private const int bar = 15;

void Baz()
{
    // Any actions with const fields
}
```

### Related information

SonarC# ver. - 5.10.1.1411
(used in codacy, possibly, it actual for newest versions)

**Comment by valhristov** [comment]
Hi @extremecodetv, I could not reproduce a false positive with the provided sample, could you please send me a complete class that can be used to observe the problem?
